pyrft/cluster_inference.py

```diff
@@ -61,17 +61,17 @@ def find_clusters(test_statistic, cdt, below = bool(0), mask = math.nan, connect
     store_cluster_sizes = np.zeros(1)
 
     # Use J to keep track of the clusters
-    J = 0
+    j = 0
 
-    for I in np.arange(n_clusters):
-        cluster_index = (cluster_image == (I+1))
+    for i in np.arange(n_clusters):
+        cluster_index = (cluster_image == (i+1))

```

Comment:
```suggestion
        cluster_index = (cluster_image == (i + 1))
``` Apparently you don't have an automatic  pep8 checker ?

Reply:
I'm confused, this was fixed last week on the testing branch. I've been waiting for you to approve the merge. Can I merge it?

Reply:
I also explained in a previous comment why I hadn't changed the I to i beforehand - it was a personal preference. But I then changed them all.